### ISSUE TITLE
Manga Livre is the best in pt-br

### ISSUE THREAD

**Issue by Irineu333** [KotatsuApp/kotatsu-parsers#92]
### Source name

Manga Livre 

### Source link

https://mangalivre.net

### Language

Português

### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have checked that the source does not already exist on the app.
- [X] I have checked that the source does not already exist by searching the [GitHub repository](https://github.com/KotatsuApp/kotatsu-parsers) and verified it does not appear in the code base.

**Comment by ztimms73** [comment]
Can't access :(
![Screenshot_20221021-082858](https://user-images.githubusercontent.com/61558546/197119722-25313a11-366a-4f34-84b9-f2f9e5c56e45.png)

**Comment by Irineu333** [comment]
![Screenshot_20221021-023301](https://user-images.githubusercontent.com/45833588/197120185-9df44fc8-839f-4f3b-80fe-2e3e9daa5fc8.png)

Maybe it's blocked by region 🤔

**Comment by Irineu333** [comment]
I can try to implement and maintain a pull request, what do you think? The bad thing is that their search uses ajax.

**Comment by ztimms73** [comment]
@Irineu333 sorry for long response. Yes, it's very cool, contributions are always welcome. You can read contribution guidelines on our repo and create draft PR, we can help if there are difficulties

**Comment by davvarrr** [comment]
Source Dead
